### PR TITLE
Upgrade to latest deploy module

### PR DESCRIPTION
**Pull request by dincho**
- add additional storage to integration environment

should fix https://www.pivotaltracker.com/n/projects/2124891/stories/167867093

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -64,14 +74,16 @@ module "aws_deploy-integration-eu-west-2" {
...
   env               = "next"
   bootstrap_version = "${var.bootstrap_version}"
   vault_role        = "ae-node"
   vault_addr        = "${var.vault_addr}"
 
-  static_nodes  = 1
-  spot_nodes    = 2
+  static_nodes   = 1
+  spot_nodes_min = 2
+  spot_nodes_max = 2
```
Comment (venimus) [inline main.tf:85]: `snapshot_filename = "mnesia_uat_v-1_latest.tgz"?

File: main.tf [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 
-  spot_nodes       = 3
-  spot_price       = "0.125"
-  instance_type    = "m4.large"
-  ami_name         = "aeternity-ubuntu-16.04-v1549009934"
-  root_volume_size = 20
+  spot_nodes_min = 3
+  spot_nodes_max = 3
+
+  spot_price    = "0.125"
```
Comment (venimus) [inline main.tf]: i guess this could be lowered as it is ~0.03-0.04

File: main.tf (hunk 1 above)
Comment (dincho) [reply main.tf:85]: next environment is not connected to the testnet

**Review by lucafavatella — DISMISSED** [review]

File: main.tf (hunk 2 above)
Comment (dincho) [reply main.tf]: https://github.com/aeternity/terraform-aws-devnet/pull/2/commits/5c1ac5e7860ea34acc76e8018fc9e2174b8a557e

**Review by lucafavatella — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (venimus) [reply main.tf:85]: ah, i didn't see it is next

**Review by venimus — APPROVED** [review]

**Pull request merged**